### DISCUSSION TITLE
Error al intentar cambiar configuración del grupo

### DISCUSSION THREAD

**Discussion by AlejandroLoraG** [Cotalker/documentation#340] [4. Bug Reports]
**Hola! Ojalá se encuentren de maravilla :)**

Cabe primero destacar que esto es en el ambiente del Cliente QA
Al intentar cambiar la configuración de un grupo desde el administrador, este cambio no queda guardado.

Adjunto pasos a seguir.

**_Editar grupo_**
![Screenshot](https://user-images.githubusercontent.com/51095635/126171768-203df527-efcd-48ff-805b-abd3f98b3c36.png)

**_Al intentar editar cualquiera de los campos del grupo, el sistema me rechaza la petición._**
![Screenshot (1)](https://user-images.githubusercontent.com/51095635/126171866-b786b77b-005a-4a85-9943-89274e3e474e.png)

¡Muchas gracias! y que sigan estando de maravilla :)

**Comment by acedward** [comment]
@AlejandroLoraG modifiqué la pregunta para eliminar nombre de clientes y endpoints del cliente que no son información pública.

**Comment by acedward** [comment]
Este error al parecer está solucionado en la última versión.

**Reply by acedward** [reply]
¿Cual era el error de red que retorna?

**Reply by AlejandroLoraG** [reply]
Codigo 400 Bad request, puedo adjuntar imagen?

**Reply by acedward** [reply]
Si, siempre y cuando no aparezcan datos privados del cliente!
